src/icons/AllSeeingSealEye.tsx

```diff
@@ -0,0 +1,67 @@
+import classnames, { stroke } from 'classnames/tailwind'
+
+const strokeSecondaryDimmed = classnames(stroke('stroke-secondary-dimmed'))
+const strokeTertiaryLine = classnames(stroke('stroke-tertiary'))
+
+export default function () {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      width="55"
+      height="46"
+      viewBox="0 0 55 46"
+      fill="none"
```

Comment:
This should be set by default from `index.css` like this:
```CSS
svg:not(.Toastify__toast-icon svg) {
  fill: none;
}
path:not(.walletconnect-qrcode__image path),
circle {
  stroke-linecap: round;
  stroke-linejoin: round;
}
```
So you'll save many lines